### ISSUE TITLE
[Concept Entry] MySQL built-in-functions

### ISSUE THREAD

**Issue by codecademy-docs** [Codecademy/docs#4677]
### Reason/inspiration (optional)

We would like a new entry on the `Built in Functions` concept under MySQL. The entry should go in a new file under `docs/content/mysql/concepts/built-in-functions/built-in-functions.md`.

The entry should include:

- An introductory paragraph that describes the concept
- A ‘Syntax’ section that includes the syntax for the concept (if applicable)
- An ‘Example’ section that shows the concept in use

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

**Comment by alimalim77** [comment]
Hi, I would like to work on this issue!

**Comment by mamtawardhani** [comment]
HEy @alimalim77! You’re assigned 🎉 In addition to the documents linked in the description, please also look at the [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide). After creating a PR, the maintainer(s) (with the collaborator label) will add comments/suggestions to address any revisions before approval.

Is this your first contribution to Codecademy Docs? If so, we’re curious to know how you found out about contributing to Docs.

**Comment by alimalim77** [comment]
Thanks for assigning, working on the issue!

**Comment by mamtawardhani** [comment]
Hey @alimalim77! Are you still working on this? Since it has been a week, could you please update me on the status of this entry? 😃

**Comment by alimalim77** [comment]
@mamtawardhani Yes, sorry for taking some time, give me another few days and I will be raising a PR for the same.

**Comment by mamtawardhani** [comment]
Hey @alimalim77, following up on the status of this entry since it has been two weeks now, please let me know the status and when can you submit this PR. Thanks 😄

**Comment by alimalim77** [comment]
Please unassign as I am busy this week. Looking forward to contribute some time in the future. Thanks!

**Comment by NelsonSanti** [comment]
Hello @mamtawardhani ,

Once @alimalim77 is busy, can you please assign it to me?
Thank you!

**Comment by mamtawardhani** [comment]
@NelsonSanti You're assigned. In addition to the documents linked in the description, please also look at our [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide) if you haven't already. After creating a PR, the maintainer(s) (with the collaborator label) will add comments/suggestions to address any revisions before approval.

Is this your first contribution to Codecademy Docs? If so, we're curious to know how you found out about contributing to Docs.

**Comment by NelsonSanti** [comment]
Thank you very much @mamtawardhani  !

Yes, this is my first contribution. I am currently a student at Codecademy taking "Learn Git & GitHub" course, at this moment in the "Contributing to the Codecademy Docs Repository" subject.

**Comment by NelsonSanti** [comment]
Hello @mamtawardhani ,

I don't know if I can ask through this channel but here I go...

I'm almost done.
I left the metadata for last, but the information seems confusing and I can't figure out how to do it.

For example, the style-guide.md says: "The Title and Description: data appear as single-quoted strings on the same line."
But the title and description appear on separate lines in the template:
"---
Title: 'The Title'
Description: 'A brief description' "

Can you please provide a visual example of how to fully complete the metadata?
Thank you

**Comment by mamtawardhani** [comment]
Hey @NelsonSanti! You can refer to [this](https://github.com/Codecademy/docs/pull/4747/files) PR for metadata, if you have any more questions, feel free to tag me 😃

**Comment by NelsonSanti** [comment]
Hello @mamtawardhani ,

Thank you for your help!
A Pull Request was created.

Thank you

**Comment by NelsonSanti** [comment]
Hello @mamtawardhani ,

Any feedback on this?
Thank you

**Comment by mamtawardhani** [comment]
> Hello @mamtawardhani ,
> 
> Any feedback on this? Thank you

Hey @NelsonSanti sorry for the inconvenience caused, I've asked the reviewer to get this reviewed ASAP.

**Comment by NelsonSanti** [comment]
> > Hello @mamtawardhani ,
> > Any feedback on this? Thank you
> 
> Hey @NelsonSanti sorry for the inconvenience caused, I've asked the reviewer to get this reviewed ASAP.

Hello @mamtawardhani ,
It is ok, I just wanted to make sure that this is not forgotten :)
Thank you
